src/modules/ui/dom.ts

```diff
@@ -508,7 +508,19 @@ export function addNoLyricsButton(song: string, artist: string, album: string, d
     window.open(url.toString(), "_blank");
   });
 
+  const searchQuery = encodeURIComponent(`${artist.trim()} - ${song}`);
```

Comment:
lets use the same logic we used before for searchQuery
```suggestion
  const searchQuery = encodeURIComponent(`${artist.trim()} - ${song.trim()}`);
```